javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +84,280 @@ private API() {
     }
 
     //
+    // Aliases for static factories
+    //
+
+    /**
+     * Alias for {@link Function0#of(Function0)}
+     *
+     * @param <R>             return type
+     * @param methodReference A method reference
+     * @return A {@code Function0}
+     */
+    public static <R> Function0<R> function0(Function0<R> methodReference) {
+        return Function0.of(methodReference);
+    }
+
+    /**
+     * Alias for {@link Function1#of(Function1)}
+     *
+     * @param <R>             return type
+     * @param <T1>            type of the 1st argument
```

Comment:
@valery1707, I still think the documentation should be updated to make it obvious that we're not referring to the wrapper method's parameters and return value. If you think it's ok this way also, I don't mind, I care more about good code than good documentation :)